server/DataBase/models/user.js

```diff
@@ -43,6 +43,7 @@ let UserSchema = new mongoose.Schema({
         courseId : {type : String},
         completed : {type : Boolean, default : false}
     }],
+    about : {type : String, default : "This is the information about me..."},
```

Comment:
Change this into " Tell us about who you are ..."

Reply:
Changed.